parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -170,7 +171,7 @@ apt_get_install() {
     for i in $(seq 1 $retries); do
         wait_for_apt_locks
         dpkg --configure -a
-        apt-get install --no-install-recommends -y ${@}
+        apt-get install -o Dpkg::Options::="--force-confold" --no-install-recommends -y ${@}
```

Comment:
"Do not modify current configuration file." Nice. I had to look that up.

Reply:
Yup had to look that up to... took me a while to figure it out :)